src/assets/wise5/components/aiChat/ai-chat-authoring/ai-chat-authoring.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+import { AbstractComponentAuthoring } from '../../../authoringTool/components/AbstractComponentAuthoring';
+
+@Component({
+  selector: 'ai-chat-authoring',
```

Comment:
Not used, remove?
```suggestion
```

Reply:
We don't remove the selector for any of the other item authoring components. Should we be consistent?